doc/wiki.txt

```diff
@@ -662,6 +663,11 @@ the commands are also available as mappings of the form `<plug>(wiki-[name])`.
   -ext EXT~
     Specify extension to use if `fname` is not provided.
 
+  -link_ext_replace~
+    Set to true to replace in-wiki link extensions with html. This enables
+    in-browser wiki navigation. Requires`ext` be set to `html` and for
+    `g:wiki_link_extension` to be non-empty to have any meaningful effect.
```

Comment:
Change `` `g:wiki_link_extension` `` to `|g:wiki_link_extension|` to enable tag based navigation.